app/representers/api/series_representer.rb

```diff
@@ -47,4 +47,14 @@ class Api::SeriesRepresenter < Api::BaseRepresenter
     }
   end
   embed :account, class: Account, decorator: Api::Min::AccountRepresenter
+
+  link :audio_version_templates do
+    {
+      href: api_series_audio_version_templates_path(represented)
```

Comment:
Can I argue to also put a `count` on this, to save me a request on the client side?  Or am I just being lazy?